src/mod.ts

```diff
@@ -1,6 +1,8 @@
 import { Context, NextFunction } from "./deps.deno.ts";
-import getEmoji from "./emoji.ts";
-import type { EmojiName } from "./emoji.ts";
+import emojis from "./emojidata.ts";
+
+type EmojiList = typeof emojis;
+type EmojiName = keyof EmojiList | (string & Record<never, never>);
```

Comment:
```suggestion
type EmojiName = keyof EmojiList | (string & Record<never, never>);
```
is clever, do you feel like opening a pull request for the core repo that switches to `Record<never, never>` for `StringWithSuggestions`? That way, we can enable linting for the respective lines.

Reply:
I don't get it. What does this improve? In the grammY core, do you also need to use `string & Record<never, never>` somewhere?

Reply:
https://github.com/grammyjs/grammY/blob/8acfe06252fb7d7b8a1f089b2446c5f117f6748c/src/context.ts#L32